config/dr_cluster/manager/ramen_manager_config.yaml

```diff
@@ -9,4 +9,5 @@ webhook:
 leaderElection:
   leaderElect: true
   resourceName: dr-cluster.ramendr.openshift.io
+  resourceNamespace: ramen-system
```

Comment:
We may need to remove this, kustomize cannot customize namespaces within, and if not required, we should not add this to the config.